extensions/hashicorp-vault/src/main/java/org/eclipse/dataspaceconnector/core/security/hashicorpvault/CreateEntryRequestPayloadOptions.java

```diff
@@ -0,0 +1,58 @@
+/*
+ *  Copyright (c) 2022 Mercedes-Benz Tech Innovation GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Mercedes-Benz Tech Innovation GmbH - Initial API and Implementation
+ *
+ */
+
+package org.eclipse.dataspaceconnector.core.security.hashicorpvault;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
If the `ObjectMapper` instantiated by the `TypeManager` is used (which it should), then this annotation is superfluous. cf. `Typemanager.java#L50`
Same goes for all the other payload classes.